app/components/search-result.js

```diff
@@ -27,6 +27,14 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    shortContributorList: Ember.computed('result', function() {
+        let filtered = this.get('result.contributors').filter(item => !!item.users.bibliographic);
+        let nameArr = filtered.slice(0, Math.min(10, filtered.length));
+        if (nameArr.length < filtered.length)
+            nameArr[nameArr.length - 1].originalLength = filtered.length;
```

Comment:
Hmm. I would still prefer tracking `hasMoreContributors` as a separate Ember.computed property. 

It's a bit confusing to use the presence or absence of a numeric property on the last item as a boolean flag, especially when the flag is controlling display of something not directly related to that item.

Here is an example (pseudocode; haven't run it yet) demonstrating the idea. It's a little annoying having so many properties, but computeds are cached, so this only runs once every time the dependent keys change- so you're not having to duplicate work to reuse logic.

search-result.js:
```
hasMoreContributors: Ember.computed('justContributors', 'shortContributorsList', function () {
  return this.get('shortContributorsList').length < this.get('justContributors').length ;
});


search-result.hbs
```
{{#each shortContributorList as |contributor|}}
  {{author-link contributor=contributor}}
{{/each}}
{{if hasMoreContributors "..."}}
```